backend/Repositories/SightingRepo.cs

```diff
@@ -19,7 +20,15 @@ public SightingRepo(WhaleSpottingDbContext context)
         {
             _context = context;
         }
-    
+
+        public Sighting CreateSighting(Sighting createSightingRequest)
+        {
+            var newSighting = _context.Sightings.Add(createSightingRequest);
+            _context.SaveChanges();
+
+            return newSighting.Entity;
```

Comment:
Variable naming here is a little strange - `createSightingRequest` here doesn't represent a request object, but the actual database model we want to insert. I think better names may be `newSighting` for the parameter, and `insertedSighting` for the retrieved entity:
```suggestion
        public Sighting CreateSighting(Sighting newSighting)
        {
            var insertedSighting = _context.Sightings.Add(newSighting);
            _context.SaveChanges();

            return insertedSighting.Entity;
```